core/src/test/java/org/elasticsearch/script/ScriptTests.java

```diff
@@ -52,6 +54,21 @@ public void testScriptParsing() throws IOException {
         }
     }
 
+    public void testScriptSimpleParsing() throws IOException {
+        String simpleScript = "{\"script\":\"_score + doc['field']\"}";
```

Comment:
Isn't this the old format we removed in 5.0? ie, not being explicit about it being an inline script.